### ISSUE TITLE
Output panels, mousewheel scroll is too fast. Let users set the speed.

### ISSUE THREAD

**Issue by HubKing** [microsoft/vscode#111264]
I have seen the existing issue https://github.com/microsoft/vscode/issues/35111 but it was closed with a comment like "I believe this has been fixed" but it does not seem to have been fixed in version 1.51.1. Also the issue has been disabled adding comment, so I could not add a comment there.

On the "TERMINAL", mouse wheel seems to scroll 4 lines, but if I adjust the panels height, sometimes one mouse wheel scrolls 13 lines. On the "DEBUG CONSOLE", one mouse wheel seems to scroll 5 lines. This is very annoying and makes me to use the scroll bar on the right, instead of the wheel.

My system setting is "3 lines" per wheel, but this is because if I set it to 1, web page scrolling becomes too slow. When browsing the web, people often need to scroll fast, but when examining outputs in VS Code, I think people often need more precise scrolling. So, instead of "using the system value", how about letting user customise the scroll speed? I would like to set "1 line per wheel".

**Comment by isidorn** [comment]
Sounds like a feature request for the terminal and the tree. Forwarding to @tyriar and @joaomoreno

**Comment by joaomoreno** [comment]
Terminal already has a setting: `terminal.integrated.mouseWheelScrollSensitivity`, maybe that helps.
List, duplicate: #102261

**Comment by HubKing** [comment]
> 
> 
> Terminal already has a setting: `terminal.integrated.mouseWheelScrollSensitivity`, maybe that helps.
> List, duplicate: #102261

I had seen only the Alt sensitivity, but after reading that, I found normal scrolling multiplier. Since this is a multiplier, not the number of lines, it was a little bit difficult to make it scroll precisely by the number of lines I want, but I set the multiplier to `0.1` to get a similar result.

However, I still have to wonder why it is still scrolled by 10+ lines when #35111 is marked as solved? See the scrolling speed with the default multiplier value 1. I slowly scrolled up/down by the minimum amount of movement.

![25-November](https://user-images.githubusercontent.com/10900489/100217169-7e719d00-2f56-11eb-9673-eeb8b5d059b0.gif)

For what it's worth, I was using the PlatformIO extension, and the output is from its upload process.
